projects/components/src/breadcrumbs/breadcrumbs.component.test.ts

```diff
@@ -6,7 +6,7 @@ import { EMPTY } from 'rxjs';
 import { BreadcrumbsComponent } from './breadcrumbs.component';
 import { BreadcrumbsModule } from './breadcrumbs.module';
 
-describe('BreadcrumbsComponent', () => {
+fdescribe('BreadcrumbsComponent', () => {
```

Comment:
please revert (also as a tip, jest lets you specify tests at the command line in watch mode (`npm run test -- --watch`)

Reply:
Reverted.